contracts/transfer/src/state.rs

```diff
@@ -352,6 +352,41 @@ impl TransferState {
             .expect("Calling receiver should succeed")
     }
 
+    /// Transfer funds from a contract balance to a Moonlight account.
+    ///
+    /// Contracts can call the function and expect that if it succeeds the funds
+    /// are successfully transferred to the contract they specify.
```

Comment:
/// Contracts can call the function and expect that if it succeeds the funds
/// are successfully transferred to the ~~contract~~ **account** they specify.

Reply:
Fixed